plugins/com.google.cloud.tools.eclipse.test.logback/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.test.logback
+Bundle-Version: 0.1.0.qualifier
+Fragment-Host: ch.qos.logback.classic
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ActivationPolicy: lazy
+Import-Package: org.slf4j.impl
+Eclipse-ExtensibleAPI: true
+Eclipse-BundleShape: dir
```

Comment:
`BundleShape: dir` tells p2 to install bundle as an exploded jar (the contents) rather than as a jar.  It means end-users can easily modify the contained `logback.xml`.  Since we're not shipping this jar, it's not so useful, but it doesn't hurt.